evap/staff/tests/test_views.py

```diff
@@ -1101,21 +1101,27 @@ def test_publish_notifications(self):
         mail.outbox = []
 
         self.helper_publish_evaluation_with_publish_notifications_for(evaluation, contributors=True, participants=False)
-        self.assertEqual(len(mail.outbox), 1)
-        self.assertEqual(mail.outbox[0].to, [contributor1.email])
+        self.assertEqual(len(mail.outbox), 2)
+        self.assertEqual(
+            {recipient for msg in mail.outbox for recipient in msg.to},
+            {contributor1.email, self.responsible.email}
+        )
         mail.outbox = []
 
         self.helper_publish_evaluation_with_publish_notifications_for(evaluation, contributors=False, participants=True)
         self.assertEqual(len(mail.outbox), 2)
-        self.assertIn([participant1.email], [mail.outbox[0].to, mail.outbox[1].to])
-        self.assertIn([participant2.email], [mail.outbox[0].to, mail.outbox[1].to])
+        self.assertEqual(
+            {recipient for msg in mail.outbox for recipient in msg.to},
+            {participant1.email, participant2.email}
+        )
         mail.outbox = []
 
         self.helper_publish_evaluation_with_publish_notifications_for(evaluation, contributors=True, participants=True)
-        self.assertEqual(len(mail.outbox), 3)
-        self.assertIn([participant1.email], [mail.outbox[0].to, mail.outbox[1].to, mail.outbox[2].to])
-        self.assertIn([participant2.email], [mail.outbox[0].to, mail.outbox[1].to, mail.outbox[2].to])
-        self.assertIn([contributor1.email], [mail.outbox[0].to, mail.outbox[1].to, mail.outbox[2].to])
+        self.assertEqual(len(mail.outbox), 4)
+        self.assertEqual(
+            {recipient for msg in mail.outbox for recipient in msg.to},
+            {participant1.email, participant2.email, contributor1.email, self.responsible.email}
+        )
```

Comment:
On a second thought: The sets will remove duplicates. Thus, the new code will not catch if we are sending emails way too often or add the wrong people as recipients. The old assertions were probably better.